app/services/TtpService.scala

```diff
@@ -155,6 +166,19 @@ class TtpService @Inject() (ttpConnector: TtpConnector, datesService: DatesServi
     )
 
     ttpConnector.callArrangementApi(arrangementRequest, journey.correlationId)
+      .map { response: ArrangementResponse =>
+        auditService.auditPaymentPlanSetUp(journey, Right(response))
+        response
+      }.recover {
+        case httpException: HttpException =>
+          try {
```

Comment:
we probably don't need a try around this audit - if we want to we could stick a try block around the `audit` method in the `AuditService` maybe. That way it will catch any audit failures - but then at that point I wouldn't necessarily throw an error - I'd just log it at warn or error level. If we can't audit then it shouldn't affect the user journey I don't think.